third_party/2and3/redis/client.pyi

```diff
@@ -281,6 +283,7 @@ class Redis(object):
     def pubsub_numsub(self, *args: Text) -> List[Tuple[Text, int]]: ...
     def pubsub_numpat(self) -> int: ...
     def monitor(self) -> Monitor: ...
+    def cluster(self, cluster_arg, *args): ...
```

Comment:
`cluster_arg` must be a `str`. `args` can be annotated with `Any`, since we can't know the exact types (as they depend on the cluster command). Same is true for the return type, so this should also be `Any` to explicitly mark it.

Reply:
about `cluster_arg` I agree, good point.

regarding the *args and return type, I understood (possibly incorrectly) the CONTRIBUTING.md as preferring unspecified parameter and return types rather than explicit `Any`s:

> Do not use Any to mark unannotated arguments or return values.
https://github.com/python/typeshed/blob/master/CONTRIBUTING.md#incomplete-stubs

Also I was copying the other annotations in the same file.

Would you prefer I used explicit `Any` in this case?

Reply:
Yes, no annotation is basically meant to show that no one has looked at the types yet, while `Any` means that someone has looked, but we can't use more precise types. The sentence from CONTRIBUTING does not explain that very well.